plugins/ilert/package.json

```diff
@@ -26,7 +26,7 @@
     "@backstage/errors": "^0.1.1",
     "@backstage/plugin-catalog-react": "^0.2.4",
     "@backstage/theme": "^0.2.8",
-    "@date-io/luxon": "1.x",
+    "@date-io/luxon": "2.x",
```

Comment:
Shouldn't this be `luxon` already? 🤔

Reply:
I mean I'm not 100% sure what the usage of this `@date-io` abstraction is, maybe we could remove it in favour of luxon?

Reply:
Let's merge for now and aim to remove later.